datajoint/blob.py

```diff
@@ -240,6 +266,9 @@ def pack_array(self, array):
         """
         Serialize an np.ndarray into bytes.  Scalars are encoded with ndim=0.
         """
+        if "datetime64" in array.dtype.name:
```

Comment:
still wondering if should encode the precision rather than default to [us]

Reply:
I will look into a way of encoding the precision